common/src/python/flywheel_adaptor/project_adaptor.py

```diff
@@ -160,3 +163,70 @@ def remove_gear_rule(self, *, rule: GearRule) -> None:
           rule: the rule to remove
         """
         self.__fw.remove_project_gear_rule(project=self.__project, rule=rule)
+
+    def get_apps(self) -> List[ViewerApp]:
+        """Returns the list of viewer apps for the project.
+
+        Returns:
+          the viewer apps for the project
+        """
+        return self.__fw.get_project_apps(self.__project)
+
+    def set_apps(self, apps: List[ViewerApp]) -> None:
+        """Sets the viewer apps for the project.
+
+        Args:
+          apps: the list of viewer apps to add
+        """
+        self.__fw.set_project_apps(project=self.__project, apps=apps)
+
+    def get_dataviews(self) -> List[DataView]:
+        """Returns the list of dataviews for the project.
+
+        Returns:
+          the dataviews in the enclosed project
+        """
+        return self.__fw.get_dataviews(self.__project)
```

Comment:
This also ends up returning all dataview templates available to the project, even if they're not actually imported or used.  You may want to filter out only ACTIVE dataviews with the following check:

```
dataviews = self.__fw.get_dataviews(self.__project)
dataviews = [view for view in dataviews is view.parent != "site"]
```